packages/amplify-category-auth/provider-utils/awscloudformation/assets/cognito-defaults.js

```diff
@@ -29,20 +29,22 @@ const roles = {
 
 const generalDefaults = projectName => ({
   resourceName: `${projectName}${sharedId}`,
+  resourceNameTruncated: `${projectName.substring(0, 6)}${sharedId}`,
   authSelections: 'identityPoolAndUserPool',
   ...roles,
 });
 
 const userPoolDefaults = (projectName) => {
   const projectNameTruncated = `${projectName.substring(0, 6)}${sharedId}`;
   return ({
+    resourceNameTruncated: `${projectName.substring(0, 6)}${sharedId}`,
     userPoolName: `${projectName}_userpool_${sharedId}`,
     autoVerifiedAttributes: ['email'],
     mfaConfiguration: 'OFF',
     mfaTypes: ['SMS Text Message'],
-    roleName: `${projectNameTruncated}_sns-role`,
-    roleExternalId: `${projectNameTruncated}_role_external_id`,
-    policyName: `${projectNameTruncated}-sns-policy`,
+    // roleName: `${projectNameTruncated}_sns-role`,
```

Comment:
Why are these commented out?

Reply:
I commented them out while testing their removal (had to decrease the number of parameters in the cf template, and these never change anyway so they aren't needed).  I'm deleting them now.  Thanks for catching this.